src/runtime_src/core/tools/xbmgmt2/XBMgmtMain.h

```diff
@@ -0,0 +1,31 @@
+/**

```

Comment:
Is this file specific to xbmgmt or is it a verbatim copy from xbutil?
It latter then lets refactor to share common code.

Reply:
It's the exact same code. I'll refactor it in the next PR